prodnet/README.md

```diff
@@ -179,15 +179,19 @@ There will be two users: `admin` with `ADMIN_PASSWORD`, and `me` with no passwor
 
 1. Connect the MikroTik device to a router with a DHCP server
 
-1. Connect your computer to the router (not to the access point of the MikroTik device, otherwise configuration scripts will not fully execute whenever a command resets the access point)
+1. Connect your computer directly to the MikroTik device using wireless (not ethernet). There should be 2 new SSIDs for it.
 
 1. Scan for the IP address of the MikroTik device with a tool like `nmap` or `arp-scan`
 
 1. SSH into the device with `ssh admin@IP_ADDRESS`
 
-1. Ensure the device has fresh configurations, run `/system reset-configuration` if needed
-
-1. Run [wap/wap-ap.rsc](wap/wap-ap.rsc) after replacing the `ADMIN_PASSWORD`
+1. Run the commands in [wap/wap-ap.rsc](wap/wap-ap.rsc) in stages:
+    1. Run the first command after replacing the `ADMIN_PASSWORD`.
+    1. Run the next commands upto and including line 54, this will change the password and break your admin connection.
```

Comment:
I think documents relying on specific line number in a script risks falls out of sync rather quickly when scripts get edited. I will merge this for now since the previous docs don't actually work 😉but will definitely do a pass later on entire doc for consistent styles after everyone have contributed their bits.

Thanks for the PR!